FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.E2ETests/TestData/Expected/Dicom_All.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "StartTime": "2022-12-01T09:18:12.0692082+00:00",
+  "CompleteTime": "2022-12-01T09:20:57.0367552+00:00",
+  "CreatedJobCount": 13,
+  "RunningJobIds": [],
+  "TotalResourceCounts": { "dicom": 1246 },
+  "ProcessedResourceCounts": { "dicom": 1246 },
+  "SkippedResourceCounts": { "dicom": 0 },
```

Comment:
"Dicom" as well?

Reply:
`dicom` was used in this PR, and modify to `Dicom` when creating a new pr to change to `Dicom`